test/controllers/CheckYourAnswersControllerSpec.scala

```diff
@@ -310,7 +311,7 @@ class CheckYourAnswersControllerSpec extends SpecBase with MockitoSugar with Bef
                 .set(CanPhoneIndividualPage, false).success.value
                 .set(SoleTraderNamePage, SoleTraderName("first", "last")).success.value
 
-              val expectedRegistrationRequest = IndividualWithoutId("first", "last", aDateOfBirth, Address.fromUkAddress(aUkAddress))
+              val expectedRegistrationRequest = IndividualWithoutId("first", "last", aDateOfBirth, Address.fromUkAddress(aUkAddress), aContact)
```

Comment:
Make the data obvious, check the changes I did in your BE commit.

Reply:
Done.